app/src/main/java/org/mozilla/focus/fragment/FragmentDispatcher.kt

```diff
@@ -0,0 +1,134 @@
+/* -*- Mode: Java; c-basic-offset: 4; tab-width: 4; indent-tabs-mode: nil; -*-
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+package org.mozilla.focus.fragment
+
+import android.content.Context
+import android.support.v4.app.FragmentManager
+import android.text.TextUtils
+import org.mozilla.focus.R
+import org.mozilla.focus.session.SessionManager
+import org.mozilla.focus.session.Source
+import org.mozilla.focus.telemetry.TelemetryWrapper
+import org.mozilla.focus.telemetry.UrlTextInputLocation
+import org.mozilla.focus.utils.OnUrlEnteredListener
+import org.mozilla.focus.utils.UrlUtils
+import org.mozilla.focus.widget.InlineAutocompleteEditText
+
+class FragmentDispatcher {
+
+    companion object {
```

Comment:
nit: if a class contains no member var/fn, you can actually make it top-level object and make "static" calls without a companion object:
```kotlin
object FragmentDispatcher {
    ...
```